.github/workflows/ci.yml

```diff
@@ -232,7 +232,7 @@ jobs:
           shell: bash
           environment_variables: AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY AWS_DEFAULT_REGION AWS_REGION
           run: |
-            sudo pkg install -y python3 py39-urllib3 py39-pip py39-awscli cmake
+            sudo pkg install -y python3 devel/py-pip net/py-urllib3 devel/py-awscli cmake
```

Comment:
did `install python3` change from being 3.9 to 3.11?

does `python3` point to whatever is "latest"? and like `devel/py-pip` points to whatever is "latest"?

If so, this seems fine

Reply:
not sure if they just updated recently or not. but the current default version for python is 3.11 https://cgit.freebsd.org/ports/tree/Mk/bsd.default-versions.mk#n139

https://wiki.freebsd.org/Python looks like they will use the default version for all the python packages.

Reply:
https://github.com/freebsd/freebsd-ports/commit/62a23ef0936e21fef507653054d6732bb7ef085f oh, it was updated 2 month ago.